src/tagmanager/tm_source_file.c

```diff
@@ -84,7 +84,7 @@ static int get_path_max(const char *path)
 }
 
 
-#ifdef G_OS_WIN32
+#if defined(G_OS_WIN32) && !defined(HAVE_REALPATH)
```

Comment:
You could probably drop the `G_OS_WIN32` part altogether, but this is probably safer for now.

Reply:
Well, the implementation only compiles on Windows.